content/en/docs/concepts/security/windows-security.md

```diff
@@ -0,0 +1,53 @@
+---
+reviewers:
+- jayunit100
+- jsturtevant
+- marosset
+- perithompson
+title:    Security For Windows Nodes
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+This page describes security considerations and best practices specific to the Windows operating system.
+
+<!-- body -->
+## Protection for Secret data on nodes
+
+On Windows, data from Secrets are written out in clear text onto the node's local
+storage (as compared to using tmpfs / in-memory filesystems on Linux). As a cluster
+operator, you should take both of the following additional measures:
+
+1. Use file ACLs to secure the Secrets' file location.
+1. Apply volume-level encryption using [BitLocker](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-how-to-deploy-on-windows-server).
+
+## Container Users
+
+[RunAsUsername](/docs/tasks/configure-pod-container/configure-runasusername)
+can be specified for Windows Pods or containers to execute the container
+processes as specific user. This is roughly equivalent to
+[RunAsUser](/docs/concepts/policy/pod-security-policy/#users-and-groups).
+
+Windows containers offer two default user accounts, ContainerUser and ContainerAdministrator.
+The differences between these accounts are covered [here](https://docs.microsoft.com/virtualization/windowscontainers/manage-containers/container-security#when-to-use-containeradmin-and-containeruser-user-accounts).
+
+Local users can be added to container images during the container build process.
+
+{{< note >}}
+
+* NanoServer based images run as `ContainerUser` by default
+* ServerCore based images run as `ContainerAdministrator` by default
```

Comment:
It's “Server Core” isn't it? I'd hyperlink to https://docs.microsoft.com/en-us/windows-server/administration/server-core/what-is-server-core

Reply:
That server core is different from the server core container image.
That one topic describes an installation type for the host OS intended for headless server installations.

I will link to https://capz.sigs.k8s.io/developers/development.html#conformance-testing instead tho!

Reply:
Added a link